sw/device/tests/dif/dif_uart_sanitytest.c

```diff
@@ -0,0 +1,70 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_uart.h"
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/runtime/hart.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"  // Generated.
+
+extern bool uart_reconfigure_required;
```

Comment:
Consider putting this in `test_main.h` instead. Summoning symbols out of the aether should be avoided.

Reply:
Done in #2457 .